configs/examples/gift-adversary-experiment.cfg

```diff
@@ -0,0 +1,12 @@
+@include ../active-experiment.cfg
```

Comment:
I find it odd that this includes `active-experiment.cfg` rather than the other way around, but it seems consistent with how we do it in e.g. `iterated-training/alternate-experiment.cfg` so I guess that's working as intended.